projects/igniteui-angular/src/lib/core/styles/components/snackbar/_snackbar-theme.scss

```diff
@@ -95,6 +104,7 @@
         background: --var($theme, 'background');
         backface-visibility: hidden;
         box-shadow: --var($theme, 'shadow');
+        border-radius: --var($theme, 'border-radius')
```

Comment:
Remove any previous border-radius declaration to fix linting errors thrown during build.